nzedb/Categorize.php

```diff
@@ -1463,7 +1639,9 @@ public function isMusicOther()
 			}
 
 			return true;
-		} else if (preg_match('/\(pure_fm\)|-+\(?(2lp|cd[ms]([-_ .][a-z]{2})?|cover|ep|ltd_ed|mix|original|ost|.*?(edit(ion)?|remix(es)?|vinyl)|web)\)?-+((19|20)\d\d|you$)/i', $this->releaseName)) {
```

Comment:
Please revert changes to this method, as it needs to be refactored to remove the excessive returns.